lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -453,6 +478,141 @@ def restart_services(self, services):
         """
         self._magmad_client.restart_services(services)
 
+    def create_mme_config_backup(self):
```

Comment:
function name "create_backup_or_restore_mme_config" would be more descriptive.

Reply:
Modified the function name and updated the PR